### PR TITLE
Fixes #2631: Remove Travis config and references

### LINKED ISSUE #2631: Remove Travis?
In #2624, I've got the tests running on CircleCI. And Travis is failing to complete the Itern tests, while Circle CI is working.

Should we ditch Travis? Seems like noise at this point.

### PR DESCRIPTION
**Pull request by miketaylr**
r? @magsout

### CODE REVIEW

File: docs/tests.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ npm run test:python
 
 Running functional tests is a bit more involved. You can also run both test suites at once (see the next section).
 
-Tests are also run automatically on [Travis](https://travis-ci.org/webcompat/webcompat.com) for each commit. If you would like to skip running tests for a given commit, you can use use the magical `[ci skip]` string in your commit message. See the [Travis docs](http://docs.travis-ci.com/user/how-to-skip-a-build/#Not-All-Commits-Need-CI-Builds) for more info.
+Tests are also run automatically on [Cirlce CI](https://circleci.com/gh/webcompat/webcompat.com) for each commit. If you would like to skip running tests for a given commit, you can use use the magical `[ci skip]` string in your commit message. See the [Circle CI docs](https://circleci.com/docs/2.0/skip-build/) for more info.
```
Comment (magsout) [inline docs/tests.md]: little typo : [Cirlce CI] -> [Circle CI]

**Comment by magsout** [comment]
@miketaylr not related for this PR, but maybe we need ton setup something on the repo to have this `Required` thing on circleCi 

capture d ecran 2018-10-02 a 17 27 19

**Comment by magsout** [comment]
@miketaylr I have to use my super power to confirm the merge 

capture d ecran 2018-10-02 a 17 29 36

**Comment by miketaylr** [comment]
" ci/circleci: build " should be required now!

**Pull request merged**